pages/catalog-page.ts

```diff
@@ -0,0 +1,24 @@
+import { BasePage } from "../pages/basePage";
+import { By, WebDriver } from "selenium-webdriver";
+import * as consts from "../consts";
+
+export const elements = {
+  headphonesLink: () =>
+    By.xpath("//li/a[contains(text(), 'Наушники и гарнитуры')]"),
+  headphonesTitle: () =>
+    By.xpath("//h1[contains(text(), 'Наушники и гарнитуры')]"),
+  catalogLink: () => By.className("b-main-navigation__text"),
+};
+
+export class CatalogPage extends BasePage {
```

Comment:
CatalogPage isn't used anywhere. Please, delete.

export const elements can be **moved** to `BasePage` as **public** field of class `BasePage`

Reply:
As a result - whole file will be deleted